docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Using Aspire Dashboard with Python Applications
+description: How to use the Aspire Dashboard in a Python application.
+ms.date: 08/27/2024
+ms.topic: tutorial
+ms.author: aapowell
+---
+
+# Tutorial: Using the Aspire Dashboard with Python Applications
+
+The [Aspire dashboard](overview.md) provides a great UI for viewing telemetry, and is available as a standalone container image that can be used with any OpenTelemetry-enabled app. In this article, you'll learn how to:
```

Comment:
In docs, we need to always say ".NET Aspire" never just "Aspire", per marketing.

```suggestion
The [.NET Aspire dashboard](overview.md) provides a great user experience for viewing telemetry, and is available as a standalone container image that can be used with any OpenTelemetry-enabled app. In this article, you'll learn how to:
```